Author: dererk
Description: Even though upstream has implemented a block of code to perform this check, as Tor's binary is located in /usr/sbin path, regular users are likely to lack that path in their PATH enviroment variable. This patch purpose is setting it to the right Debian's Tor package.
--- a/src/vidalia/config/TorSettings.cpp	2009-12-17 21:37:40.000000000 -0800
+++ b/src/vidalia/config/TorSettings.cpp	2009-12-17 21:38:17.000000000 -0800
@@ -68,7 +68,8 @@
   else
     setDefault(SETTING_TOR_EXECUTABLE, programFiles + "\\Tor\\tor.exe");
 #else
-  setDefault(SETTING_TOR_EXECUTABLE, "tor");
+  /** Debian patch for proper tor binary path */
+  setDefault(SETTING_TOR_EXECUTABLE, "/usr/sbin/tor");
 #endif
 
   setDefault(SETTING_TORRC,         Vidalia::dataDirectory() + "/torrc");
